src/connection/WalletConnectV2.ts

```diff
@@ -63,3 +64,37 @@ export class WalletConnectV2 extends WalletConnect {
     return super.activate(chainId)
   }
 }
+
+// Custom class for Uniswap Wallet specific functionality
+export class UniwalletConnect extends WalletConnectV2 {
```

Comment:
Why is this new class necessary? Why didn't it exist before?

Reply:
it did, but it was in the WalletConnect v1 file which has since been deleted. This is the new one (w/ dedicated analytics) that uses v2.